storage/plugins/archives/ArchivesPlugin.php

```
@@ -5224,6 +5271,241 @@ public function iiifCollectionAction(
+    // heavy lifting lives in RicJsonLdBuilder (pure, namespaced helper
+    // class) so this method stays small and PHPStan-friendly.
+    //
+    // RiC-O: https://www.ica.org/standards/RiC/ontology
+
+    /**
+     * GET /archives/{id}/ric.json — RiC-O JSON-LD for one archival_unit.
+     *
+     * Public route (no auth). Mirrors the visibility of /archives/{id}/dc.xml.
+     */
+    public function ricUnitAction(
+        ServerRequestInterface $request,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)